adoc/chapters/programming_interface.adoc

```diff
@@ -17736,11 +17736,9 @@ Where [code]#OP# is: [code]#&#, [code]#|#, [code]#^#.
 a@
 [source]
 ----
-marray<RET, NumElements> operatorOP(const DataT& lhs, const marray& rhs)
+marray<bool, NumElements> operatorOP(const DataT& lhs, const marray& rhs)
 ----
-   a@ Available only when: [code]#DataT != float && DataT != double && DataT != half#.
-
-Construct a new instance of the [code]#marray# class template with [code]#DataT = bool# and
+   a@ Construct a new instance of the [code]#marray# class template with [code]#DataT = bool# and
 same NumElements as [code]#lhs# [code]#marray# with each element of the new [code]#marray# instance
```

Comment:
```suggestion
same NumElements as [code]#rhs# [code]#marray# with each element of the new [code]#marray# instance
```

This also looks like it was wrong in the description.

Reply:
Thanks, fixed it and also another occurrence of the same problem.